src/lib/stores/chain-config.ts

```diff
@@ -27,7 +26,7 @@ export class ChainConfigStore {
   }
 
   getChainConfig(chainId: string): Option<SharedChainConfig> {
```

Comment:
Just `SharedChainConfig`

Reply:
if `chainId` is not in the record, the return value of `chainConfigs[chainId]` is undefined